db_stress_tool/db_stress_test_base.h

```diff
@@ -412,9 +412,8 @@ class StressTest {
   std::atomic<bool> db_preload_finished_;
   std::shared_ptr<SstQueryFilterConfigsManager::Factory> sqfc_factory_;
 
-  // Fields used for continuous verification from another thread
```

Comment:
I guess the original others intended for `cmp_db_` to potentially be used for other purposes, but right now the only usages are for opening secondary databases. So I think we can improve the naming